crates/typos-cli/src/bin/typos-cli/main.rs

```diff
@@ -32,6 +32,13 @@ fn run() -> proc_exit::ExitResult {
 
     init_logging(args.verbose.log_level());
 
+    // https://github.com/console-rs/dialoguer/issues/294
```

Comment:
It would help to put in a bit more detail, like
```rust
// HACK: Ensure the terminal gets reset to a good state if the user hits ctrl-c during a prompt
// see https://github.com/console-rs/dialoguer/issues/294 for more details
```